Realm/RLMObjectBase.mm

```diff
@@ -289,3 +269,31 @@ - (NSUInteger)hash {
 }
 
 @end
+
+
+
+Class RLMObjectUtilClass(bool isSwift) {
+    static Class objectUtilObjc = [RLMObjectUtil class];
+    static Class objectUtilSwift = NSClassFromString(@"RealmSwift.ObjectUtil");
+    return isSwift && objectUtilSwift ? objectUtilSwift : objectUtilObjc;
```

Comment:
objectUtilSwift doesn't exist for the old swift apis - this is how we fall back to the objc version.